src/library/Uno.Material/Converters/FirstCharacterConverter.cs

```diff
@@ -1,31 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-#if WinUI
-using Microsoft.UI.Xaml.Data;
-#else
-using Windows.UI.Xaml.Data;
-#endif
-
-namespace Uno.Material
-{
-	public class FirstCharacterConverter : IValueConverter
```

Comment:
Isn't this a breaking change? Are we okay with that?

Reply:
I see you're documenting that in #1326.

Though the scale of the breaking change is quite scary. Is this going to be in a major version? I think such break in a minor version will be too much.

Reply:
Yes will be a major version. @kazo0 is aware of it and has ever everything planned. If you see, this PR is just a merge from feature to master, the team has reviewed this code before.